services/daily-log-export-service.tsx

```
@@ -18,71 +37,135 @@ export class DailyLogExportService {
-
-      URL.revokeObjectURL(url)
+      this.downloadFromUrl(url, `daily-log-${dailyLog.user}-${dailyLog.date.replace(/\//g, "-")}.pdf`)
     } catch (error) {
       console.error("PDF download error:", error)
       throw error
     }
   }
 
-  static async exportToCsv(dailyLog: any, careEvents: any[]): Promise<void> {
+  static async exportToCsv(
+    dailyLog: any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)